Codeinwp/neve
# chore: added filter for the discount welcome discount Codeinwp/neve-p…

### Summary

Add the filter for the welcome discount

### Will affect the visual aspect of the product

NO

### Screenshots 

### Test instructions


1. Test that when Neve Pro is not active and the theme is installed for more than 7 days the notice is displayed
2. Check that the upgrade link redirects correctly and when trying to checkout the code is applied

To update the theme install to 7 days earlier use this command:
```
wp option update neve_install <unix_time_in_seconds>
```

## Check before Pull Request is ready:



Closes Codeinwp/neve-pro-addon#2503.
